src/cliente/js/abstractos.js

```diff
@@ -1,5 +1,47 @@
+class CacheJSONs {
+    constructor() {
+        this.jsonsEnCache = [];
+    }
+    getJSON(url) {
+        var diferido = new $.Deferred();
+        var filtrados = this.jsonsEnCache.filter(this.filtrarJson.bind(this, url))
```

Comment:
Esto se podría simplificar con 
```suggestion
        var filtrados = this.jsonsEnCache.filter(buscando => this.filtrarJson(url, buscando));
```
(que si, es una función llamando a otra, pero vamos, con lo que me ha costado entender porque has usado bind ahi si no usas this... no merece la pena)
e incluso llamar `buscando` -> `cacheado` o algo que se entienda mejor, al principio entendí que esa era la url solicitada.

`filtrados` también me chirria un poco, llamarlo `coincidencias` quizas; estaba leyendo el splice de abajo y era como "que era eso de filtrado?"